website/client/assets/scss/dropdown.scss

```diff
@@ -40,10 +40,31 @@
     background-color: inherit;
   }
 
-  &:active, &:hover, &.active {
+  &:active, &:hover {
     background-color: rgba(#d5c8ff, 0.32);
     color: $purple-200;
   }
+
+  &.active {
+    background-color: inherit;
+    color: $purple-200;
+    
+    &:active, &:hover, &:focus {
+      background-color: rgba(#d5c8ff, 0.32);
```

Comment:
can you change this to use a color variable from scss/colors.scss? thanks

Reply:
Is that `$suggested-item-color`? Or should it be something in the `$purple-XXX` family?